lib/ex_unit/test/ex_unit/callbacks_test.exs

```diff
@@ -10,7 +10,16 @@ defmodule ExUnit.CallbacksTest do
       use ExUnit.Case
 
       setup_all do
-        {:ok, [context: :setup_all]}
+        {:ok, [context: :setup_all, value: 1]}
+      end
+
+      setup_all %{context: :setup_all, value: value} = context when is_integer(value) do
+        assert value == 1
+        {:ok, context}
+      end
+
+      setup_all %{value: value} when is_binary(value) do
+        flunk "value is not a binary, I should not be executed"
```

Comment:
I think both `flunk`s should be consistent in the capitalization of the first letter, but it looks like in the codebase there's no clear winner; I'd go with lowercase for now.